tests/jobs/auto_correlate_observations/unit/test_auto_correlate_observations.py

```diff
@@ -0,0 +1,239 @@
+import configsuite
+import itertools
+import numpy as np
+import pandas as pd
+import pytest
+import random
+
+from semeio.jobs import auto_correlate_observations
+
+
+class MockedMeasuredData(object):
```

Comment:
Could you make a comment on the data we mock here?

Reply:
We mock the `MeasuredData` object in ERT. Would you like me to put a reference to it?